mirrord/operator/src/crd/policy.rs

```diff
@@ -104,9 +110,26 @@ pub struct EnvPolicy {
     /// Variable names can be matched using `*` and `?` where `?` matches exactly one occurrence of
     /// any character and `*` matches arbitrary many (including zero) occurrences of any character,
     /// e.g. `DATABASE_*` will match `DATABASE_URL` and `DATABASE_PORT`.
+    #[serde(default)]
     pub exclude: HashSet<String>,
 }
 
+#[derive(Clone, Default, Debug, Deserialize, Eq, PartialEq, Serialize, JsonSchema)]
+#[serde(rename_all = "kebab-case")]
+pub struct FsPolicy {
+    #[serde(default)]
+    pub read_only: HashSet<String>,
+
+    #[serde(default)]
+    pub read_write: HashSet<String>,
+
+    #[serde(default)]
+    pub local: HashSet<String>,
```

Comment:
What would local do? client will see that and fallback to local?

Reply:
That's a good idea, but not what I had in mind.

I thought that `local` here would mean that this file has to be read locally, so the operator returns `NotFound` if the user does not put it under `config.fs.local`.

Returning a `OpenLocal` (or something like that) and having mirrord turn this `open` call into a local one would be neat. Should I do this?

Reply:
I'd assume the notfound to be returned on the next proeprty (not_found) - but yeah I like that idea:)